.github/workflows/release.yml

```diff
@@ -1,158 +1,206 @@
 name: release
 
+on:
+  workflow_dispatch:
+    inputs:
+      dry_run:
+        description: "Run the release without actually releasing bits"
+        type: boolean
+        default: true
+      crate:
+        description: "The crate to release. (containerd-shim-wasmtime)"
+        type: string
+      version:
+        description: "The version of the crate to release. (e.g., 1.2.3)"
+        type: string
+
 concurrency:
-  group: release-${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  group: release-${{ github.workflow }}-${{ inputs.crate }}-${{ inputs.version }}
   cancel-in-progress: true
 
-on:
-  push:
-    tags:
-      - '**/v[0-9]+.[0-9]+.*'
-
 env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  parse:
-    runs-on: ubuntu-latest
-    name: Parse ref
+  pre-release:
+    name: pre-release checks
+    runs-on: "ubuntu-latest"
     outputs:
-      crate: ${{ steps.parse.outputs.crate }}
-      version: ${{ steps.parse.outputs.version }}
-      runtime: ${{ steps.parse.outputs.runtime }}
+      crate: ${{ inputs.crate }}
+      runtime: ${{ steps.runtime_sub.outputs.runtime }}
+      version: ${{ inputs.version }}
+      ### is_shim is a string, not a boolean, so use: is_shim == 'true'
+      is_shim: ${{ steps.runtime_sub.outputs.is_shim }}
     steps:
-      - uses: actions/checkout@v3
+      # - name: Fail if branch is not main
+      #   if: github.event_name == 'workflow_dispatch' && github.ref != 'refs/heads/main'
+      #   run: |
+      #     echo "This workflow should not be triggered with workflow_dispatch on a branch other than main"
+      #     exit 1
```

Comment:
does this need to be re-enabled?

Reply:
Yes! This was intentional until we are ready to merge for testing purposes.